### PR TITLE
Fix miscellaneous typos and emojis

### PR DESCRIPTION
**Pull request by charislam**
# Description

[Short summary of why you created this PR]

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

File: timescaledb/contribute-to-docs.md [hunk 1]
```diff
@@ -86,16 +86,16 @@ that case, feel free to ignore them.
 *Adverbs*
 : Do not use.
 
-:+1: Install TimescaleDB.
+&#128077; Install TimescaleDB.
```
Comment (Loquacity) [inline timescaledb/contribute-to-docs.md:89]: @jacobprall What's best practice here? I thought we discussed using the HTML codes previously and Gatsby broke them, but they seem to be working now?

Comment (charislam) [reply timescaledb/contribute-to-docs.md:89]: Happy to do these any way we agree on but the Github-style ones don't currently display as emojis :) Unless the decision was that we'd prefer to leave them as plaintext?

Comment (jacobprall) [reply timescaledb/contribute-to-docs.md:89]: Interesting. Not seeing any errors in the logs for this deploy, and they appear to be working on staging - no reason to think they wouldn't work in prod. I'd say the html codes are the way to go - but I will keep an eye on deploys to make sure they don't break any deploys

**Review by jacobprall — APPROVED** [review]

File: timescaledb/contribute-to-docs.md (hunk 1 above)
Comment (Loquacity) [reply timescaledb/contribute-to-docs.md:89]: Thanks Jacob.

**Pull request merged**